.github/workflows/deploy.yaml

```
@@ -0,0 +1,70 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: "npm"
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+        env:
+          GITHUB_USERNAME: ${{ vars.GITHUB_USERNAME }}
```

actionlint (1.7.12)

[error] configuration variable name "github_username" must not start with the GITHUB_ prefix (case insensitive). note: see the convention at https://docs.github.com/en/actions/learn-github-actions/variables#naming-conventions-for-configuration-variables

(expression)